tests/mlflow/h2o/test_h2o_model_export.py

```diff
@@ -0,0 +1,69 @@
+# pep8: disable=E501
+
+from __future__ import print_function
+
+import collections
+import os
+import pandas
+import shutil
+import unittest
+
+import sklearn.datasets as datasets
+import h2o
+from h2o.estimators.gbm import H2OGradientBoostingEstimator
+
+import tempfile
+import mlflow.h2o
+from mlflow import pyfunc
+from mlflow import tracking
+from mlflow.utils.file_utils import TempDir
+
+
+class TestModelExport(unittest.TestCase):
+    def setUp(self):
+        h2o.init()
+        iris = datasets.load_iris()
+        data = h2o.H2OFrame({
+            'feature1': list(iris.data[:, 0]),
+            'feature2': list(iris.data[:, 1]),
+            'target': list(map(lambda i: "Flower %d" % i, iris.target))
+        })
+        train, self.test = data.split_frame(ratios=[.7])
+
+        self.gbm = H2OGradientBoostingEstimator(ntrees=10, max_depth=6)
+        self.gbm.train(['feature1', 'feature2'], 'target', training_frame=train)
+        self.predicted = self.gbm.predict(self.test).as_data_frame()
+
+    def test_model_save_load(self):
+        with TempDir(chdr=True, remove_on_exit=True) as tmp:
+            path = tmp.path("model")
+            mlflow.h2o.save_model(self.gbm, path)
+
+            # Loading h2o model
+            print(path)
```

Comment:
These prints might be useful for debugging, but let's remove them for now (if someone's debugging this individual test they can add prints as needed)